course/src/main/java/in/testpress/course/ui/MeetingScreenFragment.kt

```diff
@@ -35,9 +45,53 @@ class MeetingScreenFragment : Fragment(), MeetingShareCallback.ShareEvent {
         super.onViewCreated(view, savedInstanceState)
         primaryVideoViewManager = meetingScreenBinding.primaryMeetingView.videoViewManager
         webCamVideoViewManager = meetingScreenBinding.webCamView.videoViewManager
+        audioController = inMeetingService.inMeetingAudioController
+        optionBarFragment = meetingScreenBinding.optionBar.getFragment<MeetingOptionBarFragment>()
+        setCallbackOnOptionBar()
         renderVideo()
     }
 
+    private fun setCallbackOnOptionBar(){
+        optionBarFragment?.setCallback(object: MeetingOptionBarFragment.Companion.MeetingOptionBarCallback {
+            override fun onClickChats() {
+                val sidebar = meetingScreenBinding.sidebar
+                if (sidebar.isVisible){
+                    meetingScreenBinding.sidebar.visibility = View.GONE
+                }else{
+                    meetingScreenBinding.sidebar.visibility = View.VISIBLE
+                }
+                renderVideo()
```

Comment:
Why is this here?

Reply:
To rerender the video with the allocated size